### DISCUSSION TITLE
How do I get the title of a localized field of a class?

### DISCUSSION THREAD

**Discussion by marvingoossens** [pimcore/pimcore#16052] [Q&A / Support] [answered]
I know how to get the title of a not localized field:
```
$article = DataObject\Article::getById($id);
$title = $article->getClass()->getFieldDefinition("articlenumber")->getTitle();
```

But how do I get the title if the field is a localized field?

**Comment by dpfaffenbauer** [comment, marked as answer]
```$article->getClass()->getFieldDefinition('localizedfields')->getFieldDefinition('articlenumber')->getTitle()```

not tested, but I think something like this.

**Reply by marvingoossens** [reply]
Thank you! I thought I tested this, but maybe I just don't run this code because my IDE marks this as wrong. But It works :)
